L1Trigger/Phase2L1ParticleFlow/python/l1ctLayer1_cff.py

```diff
@@ -8,6 +8,11 @@
 from L1Trigger.Phase2L1ParticleFlow.l1tPFClustersFromHGC3DClusters_cfi import l1tPFClustersFromHGC3DClusters
 
 from L1Trigger.Phase2L1ParticleFlow.l1TkEgAlgoEmulator_cfi import tkEgAlgoParameters,tkEgSorterParameters
+from L1Trigger.VertexFinder.l1tVertexProducer_cfi import l1tVertexProducer
```

Comment:
Is this actually used in here?

Reply:
Thanks for the catch: this was left over from a previous iteration of the code.